node/src/test/java/ch/prevo/open/node/services/AdapterDataValidationServiceTest.java

```diff
@@ -43,10 +43,10 @@ public void testCorrectCommencementIsValid() {
         EmploymentCommencement validCommencement = new EmploymentCommencement(validEmploymentInfo, validCapitalTransferInformation);
 
         // when
-        boolean isValid = adapterDataValidationService.isValidEmploymentCommencement(validCommencement);
+        final Set<ConstraintViolation<EmploymentCommencement>> violations = adapterDataValidationService.getEmploymentCommencementViolations(validCommencement);
```

Comment:
Since we know now the constraints that are violated, I would rewrite the tests so that they're expecting exactly this single property in the failing constraints.

Reply:
Good idea, I create a new issue for this.

Reply:
👍